### PR TITLE
[zh-cn]: sync translation for Vibration API

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the #19537

### CODE REVIEW

File: files/zh-cn/web/api/vibration_api/index.md [hunk 1]
```diff
@@ -1,72 +1,80 @@
...
+navigator.vibrate(200);
+navigator.vibrate([200]);
 ```
 
 以上两个例子都可以使设备振动 200 ms.
 
 ### 振动模式
 
-一组数值描述了设备振动并且不振动的交替时间段。数组中的每个值都将转换成一个整数，然后交替解释为设备应该振动的毫秒数和不振动的毫秒数。例如：
+一组数值描述了设备振动和不振动的交替周期。数组中的每个值都将转换成一个整数，然后交替解释为设备应该振动的毫秒数和不振动的毫秒数。例如：
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/vibration_api/index.md:29]:
```suggestion
一组数值描述了设备振动和不振动的交替周期。数组中的每个值都将转换成一个整数，然后交替解析为设备应该振动的毫秒数和不振动的毫秒数。例如：
```

File: files/zh-cn/web/api/vibration_api/index.md [hunk 2]
```diff
@@ -1,72 +1,80 @@
...
+title: 振动 API
 slug: Web/API/Vibration_API
+l10n:
+  sourceCommit: 77915a2ad318fb78b1d02a35d6b1de61ea1b1f5f
 ---
 
 {{DefaultAPISidebar("Vibration API")}}
 
-大多数现代移动设备包括振动硬件，其允许软件代码通过使设备摇动来向用户提供物理反馈。Vibration API 为 Web 应用程序提供访问此硬件（如果存在）的功能，如果设备不支持此功能，则不会执行任何操作。
+大多数现代移动设备包括振动硬件，其允许软件代码通过使设备摇动来向用户提供物理反馈。**振动 API**（Vibration API）为 Web 应用程序提供访问此硬件的功能（如果存在），如果设备不支持此功能，则不会执行任何操作。
```
Comment (yin1999) [inline files/zh-cn/web/api/vibration_api/index.md:10]:
```suggestion
大多数现代移动设备包括振动硬件，其允许软件代码通过使设备振动来向用户提供物理反馈。**振动 API**（Vibration API）为 Web 应用程序提供访问此硬件的功能（如果存在），如果设备不支持此功能，则不会执行任何操作。
```

File: files/zh-cn/web/api/vibration_api/index.md [hunk 3]
```diff
@@ -1,72 +1,80 @@
...
+你可以通过指定单个值或仅由一个值组成的数组来向振动硬件发出一次脉冲：
 
 ```js
-window.navigator.vibrate(200);
-window.navigator.vibrate([200]);
+navigator.vibrate(200);
+navigator.vibrate([200]);
 ```
 
 以上两个例子都可以使设备振动 200 ms.
```
Comment (yin1999) [inline files/zh-cn/web/api/vibration_api/index.md:25]:
```suggestion
以上两个例子都可以使设备振动 200 ms。
```

File: files/zh-cn/web/api/vibration_api/index.md [hunk 4]
```diff
@@ -1,72 +1,80 @@
...
 
-一组数值描述了设备振动并且不振动的交替时间段。数组中的每个值都将转换成一个整数，然后交替解释为设备应该振动的毫秒数和不振动的毫秒数。例如：
+一组数值描述了设备振动和不振动的交替周期。数组中的每个值都将转换成一个整数，然后交替解析为设备应该振动的毫秒数和不振动的毫秒数。例如：
 
 ```js
-window.navigator.vibrate([200, 100, 200]);
+navigator.vibrate([200, 100, 200]);
 ```
 
 这会使设备振动 200 ms，然后暂停 100 ms，然后再次振动设备 200 ms。
```
Comment (yin1999) [inline files/zh-cn/web/api/vibration_api/index.md:35]:
```suggestion
这会使设备振动 200 ms，然后暂停 100 ms，然后再次振动设备 200 ms。
```

**Review by yin1999 — COMMENTED** [review]
nits:

**Review by yin1999 — APPROVED** [review]

**Pull request merged**